movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.css

```diff
@@ -4,21 +4,21 @@
   max-width: 300px;
 }
 
-.labelUnClick {
+.disableGenre {
```

Comment:
Could give a specific name than this? `disableGenreSection` || `disableGenreButton`....?